packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -131,6 +131,11 @@ export class FASTAccordion extends FASTElement {
     };
 
     private activeItemChange = (event: Event): void => {
+        if (event.defaultPrevented || event.target !== event.currentTarget) {
+            return;
+        }
+
+        event.preventDefault();
```

Comment:
Can we add tests for this scenario as well?

Reply:
Added a base set of functional tests